src/asciidoc/docs/publish/customDomains.adoc

```diff
@@ -0,0 +1,69 @@
+= Using custom domains
+:page-slug: custom-domains
+:page-description: How to configure your Koji to use a custom domain.
+
+You can assign a custom domain name to your Koji after publication.
+A custom domain can have the advantage of being easier to remember, and it usually just looks better.
+Custom domains are about where you want to serve the experience.
```

Comment:
The first sentence of the intro is improved. Overall, the intro could still use a stronger explanation of the "what" and "why" of using a custom domain to serve your Kojis. Be more direct and informative, similar to how you rewrote the "testing templates" intro to provide more context. What does it mean to use a custom domain? Why would you do it? Here's an example of a well-written [explanation of subdomains for GoDaddy users](https://www.godaddy.com/help/what-is-a-subdomain-296). The topic and audience are not the same as our page, of course, but I thought it might inspire some ideas.